JNI error during execution

The following error occurred when trying to run Train from the command line, any insight would be much appreciated:

Error: A JNI error has occurred, please check your installation and try again
Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/commons/cli/ParseException
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
	at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
	at java.lang.Class.getMethod0(Class.java:3018)
	at java.lang.Class.getMethod(Class.java:1784)
	at sun.launcher.LauncherHelper.validateMainClass(LauncherHelper.java:544)
	at sun.launcher.LauncherHelper.checkAndLoadMain(LauncherHelper.java:526)
Caused by: java.lang.ClassNotFoundException: org.apache.commons.cli.ParseException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 7 more

--- a/ivy-jars.properties
+++ b/ivy-jars.properties
@@ -45,6 +45,7 @@ ivy.jar.httpcomponents-core.name=httpcore
 ivy.jar.httpcomponents-client.rev=4.5.5
 ivy.jar.httpcomponents-client.name=httpclient
 
+LIB_DIR=lib
 JARS_JACKSON=${LIB_DIR}/${ivy.jar.jackson-core.name}-${ivy.jar.jackson.rev}.jar ${LIB_DIR}/${ivy.jar.jackson-databind.name}-${ivy.jar.jackson.rev}.jar ${LIB_DIR}/${ivy.jar.jackson-annotations.name}-${ivy.jar.jackson.rev}.jar
 JARS_COMMONS_CLI=${LIB_DIR}/${ivy.jar.commons-cli.name}-${ivy.jar.commons-cli.rev}.jar
 

